end-to-end-test/tests/onboarding/onboarding.ts

```diff
@@ -0,0 +1,164 @@
+            '#app_contents > .dc-themed-scrollbars > form > .dc-input:nth-child(2) > .dc-input__field'
+        );
+
+        await this.page.waitForSelector(
+            '#app_contents > .dc-themed-scrollbars > form > .dc-input:nth-child(2) > .dc-input__field'
+        );
+        const first_input_selector =
+            '#app_contents > .dc-themed-scrollbars > form > .dc-input:nth-child(2) > .dc-input__field';
+        await this.page.click(first_input_selector, { clickCount: 3 });
+        await this.page.keyboard.press('Backspace');
+
+        await this.page.click(first_input_selector);
+
+        await this.page.type(first_input_selector, process.env.ENDPOINT!);
+        const second_input_selector =
+            '#app_contents > .dc-themed-scrollbars > form > .dc-input:nth-child(3) > .dc-input__field';
+        await this.page.click(second_input_selector, { clickCount: 3 });
+        await this.page.keyboard.press('Backspace');
+        await this.page.type(second_input_selector, process.env.APPID!);
+        if (
+            !(await this.page.locator("input[name='is_appstore_enabled']").isChecked()) &&
+            process.env.ENDPOINT_PAGE_APP_STORE === 'true'
+        ) {
+            await this.page.click('.dc-themed-scrollbars > form > div:nth-child(4) > .dc-checkbox > .dc-checkbox__box');
+        }
+        if (
+            !(await this.page.locator("input[name='show_dbot_dashboard']").isChecked()) &&
+            process.env.ENDPOINT_PAGE_DBOT_DASHBOARD === 'true'
+        ) {
+            await this.page.click('.dc-themed-scrollbars > form > div:nth-child(5) > .dc-checkbox > .dc-checkbox__box');
+        }
+        if (
+            !(await this.page.locator("input[name='is_debug_service_worker_enabled']").isChecked()) &&
+            process.env.ENDPOINT_PAGE_DEBUG_SERVICE_WORKER === 'true'
+        ) {
+            await this.page.click('.dc-themed-scrollbars > form > div:nth-child(6) > .dc-checkbox > .dc-checkbox__box');
+        }
+
+        await this.page.waitForSelector('#deriv_app > #app_contents > .dc-themed-scrollbars > form > .dc-btn--primary');
+        await this.page.click('#deriv_app > #app_contents > .dc-themed-scrollbars > form > .dc-btn--primary');
+    }
+
+    async updateServerURLAndAppIDInLocalStorage() {
+        const server = process.env.ENDPOINT;
+        const app_id = process.env.APPID;
+        await this?.signupPage?.evaluate(`localStorage.setItem('config.server_url', '${server}');`);
+        await this?.signupPage?.evaluate(`localStorage.setItem('config.app_id', '${app_id}');`);
+        await this?.signupPage?.evaluate(`window.location.reload();`);
+        const server_url = await this?.signupPage?.evaluate(() => {
+            const result = localStorage.getItem('config.server_url');
+            return Promise.resolve(result);
+        });
+        expect(server_url).toBe(process.env.ENDPOINT);
+        await suspend(10000);
+    }
+    async demoWizardHandler() {
```

Comment:
and I don't think we have this anymore. the tests are failing after this step. we are redirecting to trading hub onboarding page